Tasks/AzureContainerAppsV0/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "azure-container-apps",
+  "version": "1.0.0",
+  "description": "Azure Pipelines Task for building and deploying Azure Container Apps.",
+  "main": "azurecontainerapps.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "keywords": [],
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "@types/node": "^10.17.0",
```

Comment:
Please bump node typings package according to the [guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/migrateNode16.md#upgrading-tasks-to-node-16)

Reply:
Changed to `^16.11.39` for node 16 handler support as it's mentioend in the `migrateNode16.md` document 👍 -- thanks for the call out